src/Dropdown.php

```diff
@@ -487,13 +487,13 @@ public static function getDropdownName($table, $id, $withcomment = 0, $translate
         /** @var \DBmysql $DB */
         global $DB;
 
-        $item = getItemForItemtype(getItemTypeForTable($table));
+        $itemtype = getItemTypeForTable($table);
 
-        if (!is_object($item)) {
+        if (!is_a($itemtype, CommonDropdown::class, true)) {
```

Comment:
This seems to break some dropdowns. While finishing up the Rackable capacity I noticed the Rack dropdown on the Rack_Item form was throwing a PHP error because this method was returning a string ('&nbps;') instead of an array. We don't only use this method for dropdown classes.

Reply:
Indeed, I fixed it by checking that $itemtype is a CommonDBTM.